Tests/Test_BoardPrecharges.c

```diff
@@ -0,0 +1,21 @@
+/**
+ * Test for board contactors
+ */ 
+
+#include "common.h"
+#include "config.h"
+#include "Precharge.h"
+
+
+int main(){
+    Precharges_Init(MOTOR_PRECHARGE);
+    Precharges_Init(ARRAY_PRECHARGE);
+    while(1){
+        Precharge_Write(MOTOR_PRECHARGE, OFF);
+        Precharge_Write(ARRAY_PRECHARGE, OFF);
+        for(int i = 0; i< 999999; i++){} //~0.5 seconds
+        Precharge_Write(MOTOR_PRECHARGE, ON);
+        Precharge_Write (ARRAY_PRECHARGE, ON);
+        for(int i = 0; i< 999999; i++){} //~0.5 seconds
+    }
+}
```

Comment:
Will any of this actually compile/work? Prechargers driver got deleted based on what I can see.

Reply:
This won't compile with TEST=...

Should I rewrite all the tests?

Reply:
I don't see the need for this new test file if we're removing the Precharge driver. I wouldn't go to the hassle of rewriting all the tests, but you could mark them with a `//BROKEN BY REMOVAL OF PRECHARGE DRIVER` at the top so that we can clean up later.